shell/src/main/java/org/crsh/vfs/spi/ram/RAMURLConnection.java

```diff
@@ -43,6 +43,6 @@ public void connect() throws IOException {
 
   @Override
   public InputStream getInputStream() throws IOException{
-    return new ByteArrayInputStream(file.getBytes());
+    return new ByteArrayInputStream(file.getBytes("UTF-8"));
```

Comment:
It would be better if this was settable, rather than forcing an encoding, especially if the default charset is not desired.